src/components/Loader/People/People.tsx

```diff
@@ -0,0 +1,120 @@
+        <div className="box table-container">
+          {loading && <Loader />}
+
+          {error && !loading && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              {error}
+            </p>
+          )}
+
+          {!people.length && !loading && (
+            <p data-cy="noPeopleMessage">There are no people on the server</p>
+          )}
+
+          {!loading && (
+            <table
+              data-cy="peopleTable"
+              className="table is-striped is-hoverable is-narrow is-fullwidth"
+            >
+              <thead>
+                <tr>
+                  <th>Name</th>
+                  <th>Sex</th>
+                  <th>Born</th>
+                  <th>Died</th>
+                  <th>Mother</th>
+                  <th>Father</th>
+                </tr>
+              </thead>
+
+              <tbody>
+                {people.map(person => {
+                  const isShe = person.sex === 'f';
+
+                  return (
+                    <tr
+                      data-cy="person"
+                      key={person.name}
+                      className={
+                        location.pathname ===
+                        `/people/${person.name.toLowerCase().replaceAll(' ', '-')}-${person.born}`
+                          ? 'has-background-warning'
+                          : ''
+                      }
+                    >
+                      <td>
+                        <Link
+                          to={`/people/${person.name.toLowerCase().replaceAll(' ', '-')}-${person.born}`}
+                          className={isShe ? 'has-text-danger' : ''}
+                        >
+                          {person.name}
+                        </Link>
+                      </td>
+
+                      <td>{person.sex}</td>
+                      <td>{person.born}</td>
+                      <td>{person.died}</td>
+                      {person.motherName &&
+                      peopleNames.includes(person.motherName) ? (
+                        <td>
+                          <Link
+                            to={`/people/${person.motherName.toLowerCase().replaceAll(' ', '-')}-${people.find(body => body.name === person.motherName)?.born}`}
+                            className="has-text-danger"
+                          >
+                            {person.motherName ? person.motherName : '-'}
+                          </Link>
+                        </td>
+                      ) : (
+                        <td>{person.motherName ? person.motherName : '-'}</td>
+                      )}
+
+                      {person.fatherName &&
+                      peopleNames.includes(person.fatherName) ? (
+                        <td>
+                          <Link
+                            to={`/people/${person.fatherName.toLowerCase().replaceAll(' ', '-')}-${people.find(body => body.name === person.fatherName)?.born}`}
```

Comment:
The `replaceAll` method is not supported in all JavaScript environments. Consider using a regular expression with `replace` instead: `replace(/ /g, '-')`.